charts/linkerd2/README.md

```diff
@@ -229,6 +229,7 @@ The following table lists the configurable parameters for the Prometheus Add-On.
 | `prometheus.resources.memory.request`  | Amount of memory that the prometheus container requests                                                                                                                               ||
 | `prometheus.ruleConfigMapMounts`             | Alerting/recording rule ConfigMap mounts (sub-path names must end in `_rules.yml` or `_rules.yaml`)                                                                                   | `[]`                                 |
 | `prometheus.scrapeConfigs`             | A scrape_config section specifies a set of targets and parameters describing how to scrape them.                                                        | `[]`                                 |
+| `prometheus.sidecarContainers`         | A sidecarContainers section specifies a list of secondary containers to run in the prometheus pod e.g. to export data to non-prometheus systems | `[]`                                 |
```

Comment:
Should we call the field just `containers`, as there is nothing special about them? :thinking:

Reply:
maybe `extraContainers`?  just `containers` seems like it's suggesting that you _need_ to define something, which you don't.

Reply:
`bonusContainers`! 😄 

Does prometheus have a standardized way of referring to these processes? If so, that seems like the right pattern to follow.

Reply:
To the best of my knowledge it does not.  However, _kubernetes_ officially calls them sidecars (cf https://kubernetes.io/docs/concepts/workloads/pods/pod-overview/#how-pods-manage-multiple-containers) so I'm going to assert moderately strongly that is the terminology we should use. :)

Reply:
That's a well-reasoned point and makes sense to me. WDYT @Pothulapati?

Reply:
Yep. Makes sense for me too!